server/routes/temporary-accommodation/manage.ts

```diff
@@ -268,8 +268,8 @@ export default function routes(controllers: Controllers, services: Services, rou
   get(paths.bookings.search.active.index.pattern, bookingSearchController.index('arrived'), {
     auditEvent: 'VIEW_SEARCH_ACTIVE_BOOKINGS',
   })
-  get(paths.bookings.search.closed.index.pattern, bookingSearchController.index('departed'), {
-    auditEvent: 'VIEW_SEARCH_CLOSED_BOOKINGS',
+  get(paths.bookings.search.departed.index.pattern, bookingSearchController.index('departed'), {
+    auditEvent: 'VIEW_SEARCH_DEPARTED_BOOKINGS',
```

Comment:
Double check with @tahb on this one - are we ok with changing the vocab in our audit logs?

Reply:
Think this is fine as the 'search bookings' isn't on in prod yet. 

If it were on prod and was for a significant event which tracked a user viewing personal data then we _could_ look at asking the audit team to update old event keys.

Reply:
Tom says he is happy with this given that the changes are yet to be released to prod.

Reply:
OK, thanks Tom